script/bootstrap

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
shell. shell never changes. wonder if OperateHappy should explore less painful scripting approaches in the future.
I agree with Jonathan Blow: https://twitter.com/Jonathan_Blow/status/1216454377155653632

Reply:
I concur - on one side, `scripts to rule them all` are great and useful because you get a similar experience with all repos.

On the other side, I _really_ would love to define this on an organizational level.